type-definitions/most.d.ts

```diff
@@ -21,12 +21,59 @@ declare interface Iterable<A> {}
 
 declare type CreateGenerator<A> = (...args: Array<any>) => Generator<A|Promise<A>, any, any>;
 
+export interface Sink<A> {
+  event(time: number, value: A): void;
+  end(time: number, value: A): void;
+  error(time: number, err: Error): void;
+}
+
+export interface Task {
+  run(time: number): void;
+  error(time: number, e: Error): void;
+  dispose(): void;
+}
+
+export interface ScheduledTask {
+  task: Task;
+  run(): void;
+  error(err: Error): void;
+  dispose(): void;
+}
+
+export interface Scheduler {
+  now(): number;
+  asap(task: Task): ScheduledTask;
+  delay(task: Task): ScheduledTask;
+  periodic(task: Task): ScheduledTask;
+  schedule(delay: number, period: number, task: Task): ScheduledTask;
+  cancel(task: Task): void;
+  cancelAll(predicate: (val: any) => boolean): void;
```

Comment:
The predicate takes a Task, so I think this should be `cancelAll(predicate: (task: Task) => boolean): void`